samples/notebooks/python/third_party/weaviate-persistent-memory.ipynb

```diff
@@ -0,0 +1,657 @@
+    "  To configure a self-hosted instance with Kubernetes, follow Weaviate's [documentation](https://weaviate.io/developers/weaviate/installation/kubernetes).|\n",
+    "\n",
+    "- **Embedded** - start a weaviate instance right from your application code using the client library\n",
+    "   \n",
+    "  This code snippet shows how to instantiate an embedded weaviate instance and upload a document:\n",
+    "\n",
+    "  ```python\n",
+    "  import weaviate\n",
+    "  from weaviate.embedded import EmbeddedOptions\n",
+    "\n",
+    "  client = weaviate.Client(\n",
+    "    embedded_options=EmbeddedOptions()\n",
+    "  )\n",
+    "\n",
+    "  data_obj = {\n",
+    "    \"name\": \"Chardonnay\",\n",
+    "    \"description\": \"Goes with fish\"\n",
+    "  }\n",
+    "\n",
+    "  client.data_object.create(data_obj, \"Wine\")\n",
+    "  ```\n",
+    "  \n",
+    "  Refer to the [documentation](https://weaviate.io/developers/weaviate/installation/embedded) for more details about this deployment method."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: if you run into SSL errors when connecting to OpenAI on macOS, see this issue for a [potential solution](https://github.com/microsoft/semantic-kernel/issues/627#issuecomment-1580912248)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from typing import Tuple\n",
+    "\n",
+    "import semantic_kernel as sk\n",
+    "from semantic_kernel.connectors.ai.open_ai import (\n",
+    "    OpenAITextCompletion,\n",
+    "    OpenAITextEmbedding,\n",
+    ")\n",
+    "\n",
+    "import os"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we instantiate the Weaviate memory store. Choose ONE of the options below.\n",
+    "\n",
+    "* If you had deployed a local Weaviate instance using Docker, then your config would look something like this:\n",
+    "\n",
+    "    ```python\n",
+    "    config = weaviate_memory_store.WeaviateConfig(\n",
+    "        url=\"http://localhost:8080\"\n",
+    "    )\n",
+    "    ```\n",
+    "\n",
+    "* If you want to use embedded Weaviate, then use the following config:\n",
+    "\n",
+    "    ```python\n",
+    "    config = weaviate_memory_store.WeaviateConfig(\n",
+    "        use_embed=True\n",
+    "    )\n",
+    "    ```\n",
+    "\n",
+    "* If you're using the Saas Weaviate Cloud Services, the config is initialized in the cell below (make sure the environment variables `WEAVIATE_URL` and `WEAVIATE_API_KEY` were set in the `.env` file)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from semantic_kernel.connectors.memory.weaviate import weaviate_memory_store\n",
```

Comment:
```suggestion
    "from semantic_kernel.connectors.memory import weaviate_memory_store\n",
```
This will be updated accordingly in the next release